bsconfig.json

```diff
@@ -1,19 +1,18 @@
 {
   "name": "bucklescript-tea",
-  "version": "0.1.1",
+  "version": "0.1.2",
   "bsc-flags": [
     "-w",
     "-40",
-    "-bs-sort-imports",
-    "-bin-annot",
     "-bs-cross-module-opt"
   ],
   "package-specs": [
     "commonjs"
   ],
   "sources": [
     {
-      "dir": "src"
+      "dir": "src",
+      "public" : "all"
```

Comment:
make sure which interfaces are installed, you may want more fine grained control later